src/lib.rs

```diff
@@ -12,21 +12,13 @@
 #![deny(clippy::all)]
 #![forbid(unsafe_code)]
 
-use std::cell::RefCell;
-use std::rc::Rc;
+mod macros;
+mod renderers;
```

Comment:
Why were these definitions moved?

Reply:
I usually put modules above imports, so main/lib.rs can reference those instead of going through `crate::`. I can change it back.